managed/CounterStrikeSharp.API/Core/GlobalContext.cs

```diff
@@ -124,27 +130,149 @@ public void LoadAll()
 
             foreach (var path in filePaths)
             {
-                var plugin = new PluginContext(path);
-
+                if (path.Contains("disabled")) continue;
                 try
                 {
-                    _loadedPlugins.Add(plugin);
-                    plugin.Load();
+                    LoadPlugin(path);
                 }
                 catch (Exception e)
                 {
-                    _loadedPlugins.Remove(plugin);
                     Console.WriteLine($"Failed to load plugin {path} with error {e}");
                 }
             }
 
-            Console.WriteLine($"All managed modules were loaded. {_loadedPlugins.Count} plugins loaded.");
+            return _loadedPlugins.Count;
+        }
+
+        public void UnloadAllPlugins()
+        {
+            foreach (var plugin in _loadedPlugins)
+            {
+                plugin.Unload();
+                _loadedPlugins.Remove(plugin);
+            }
         }
 
         public PluginContext FindPluginByType(Type moduleClass)
         {
             var plugin = _loadedPlugins.FirstOrDefault(x => x.PluginType == moduleClass);
             return plugin;
         }
+
+        public void OnCSSCommand(CCSPlayerController? caller, CommandInfo info)
+        {
+            Utilities.ReplyToCommand(caller, "  CounterStrikeSharp was created and is maintained by Michael \"roflmuffin\" Wilson.\n" +
+                "  Counter-Strike Sharp uses code borrowed from SourceMod, Source.Python, FiveM, Saul Rennison and CS2Fixes.\n" +
+                "  See ACKNOWLEDGEMENTS.md for more information.", true);
+            return;
+        }
+
+        public void OnCSSPluginCommand(CCSPlayerController? caller, CommandInfo info)
+        {
+            switch (info.GetArg(1))
+            {
+                case "list":
+                {
+                    Utilities.ReplyToCommand(caller, "List of all plugins currently loaded by CounterStrikeSharp: ", true);
+                    if (_loadedPlugins.Count() == 0)
+                    {
+                        Utilities.ReplyToCommand(caller, $"  0 plugins loaded.", true);
+                    }
+                    else
+                    {
+                        int pluginCount = 1;
+                        foreach (var plugin in _loadedPlugins)
+                        {
+                            Utilities.ReplyToCommand(caller, $"  [{pluginCount}/{_loadedPlugins.Count()}]: \"{plugin.Name}\" (\"{plugin.Version}\")", true);
+                            pluginCount++;
```

Comment:
```suggestion
                        int pluginCount = 0;
                        foreach (var plugin in _loadedPlugins)
                        {
                            Utilities.ReplyToCommand(caller, $"  [{++pluginCount}/{_loadedPlugins.Count()}]: \"{plugin.Name}\" (\"{plugin.Version}\")", true);
```

Reply:
On further thought, I think maybe we should assign an autoincrementing plugin number directly to the plugin when we load it, and then store that value. That way you can call `css_plugins unload 1` to unload a plugin without necessarily having to type the full name.

Reply:
I've added this in my latest commit by storing an id in `PluginContext`. The command syntax for Id's is `css_plugins unload #<number>`